app/assets/javascripts/3dbio_viewer/src/webapp/components/network/UniProtAccessionTextArea.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import "./Network.css";
+
+interface UniProtAccessionTextAreaProps {
+    textareaValue: string;
+    onTextareaChange: (e: string) => void;
+}
+
+const UniProtAccession: React.FC<UniProtAccessionTextAreaProps> = React.memo(props => {
+    const { textareaValue, onTextareaChange } = props;
+    return (
+        <textarea
+            id="uniProtAccession"
+            name="uniProtAccession"
+            rows={5}
+            className="form-control"
+            value={textareaValue}
+            onChange={e => onTextareaChange(e.target.value)}
+        ></textarea>
```

Comment:
Small components are on the limit. Do we need a separate component or not? I think it's ok both ways.